mustermann/lib/mustermann.rb

```diff
@@ -120,7 +120,6 @@ def self.normalized_type(type)
   def self.extend_object(object)
```

Comment:
Is this code called from anywhere? It seems we want to remove the `require` in this method 🤔

Reply:
If I'm not mistaken, Sinatra's register method calls extend, which calls extend_object.

Reply:
Correct it is used [here](https://github.com/sinatra/sinatra/blob/master/lib/sinatra/base.rb#L1454)

Reply:
@zzak Do you want me to do any change? Maybe I missed the point.

Reply:
@epergo Sorry, I was just wondering out loud nothing actionable -- but I think it'd be good to test this out a bit before we release anything